pkg/pillar/cmd/downloader/space.go

```diff
@@ -42,23 +45,45 @@ func tryReserveSpace(ctx *downloaderContext, status *types.DownloaderStatus,
 }
 
 func unreserveSpace(ctx *downloaderContext, status *types.DownloaderStatus) {
+	if status.ReservedSpace == 0 {
+		return
```

Comment:
Same here.. a log statement wouldbe useful

Reply:
Sure. Done.

Reply:
Is this an error or merely that the function has no work to do?